src/Nethermind/Nethermind.Baseline.Test/BaselineTreeTrackerTests.cs

```diff
@@ -124,7 +124,7 @@ public async Task Tree_tracker_insert_leaves([ValueSource(nameof(InsertLeavesTes
 
         private async Task<(TestRpcBlockchain TestRpc, BaselineModule BaselineModule)> InitializeTestRpc(Address address)
         {
-            SingleReleaseSpecProvider spec = new(ConstantinopleFix.Instance, 1);
+            SingleReleaseSpecProvider spec = new(ConstantinopleFix.Instance, TestChainIds.NetworkId, TestChainIds.ChainId);
```

Comment:
I expected something like ChainId.MainnetChainId instead of TestChainIds.NetworkId

Reply:
NetworkId is not ChainId, that the main idea